applications/DAOsign.md

```diff
@@ -0,0 +1,185 @@
+# DAOsign
+
+- **Team Name:** DAOsign 
+- **Payment Address:** 13s3c37rwrZfa8WQcZPdqE1zi1hGAguf2TWfWMr8f5aLBrg
+- **[Level](https://github.com/w3f/Grants-Program/tree/master#level_slider-levels):** 3
+
+## Project Overview :page_facing_up:
+
+
+### Overview
+
+DAOsign is a decentralized platform for agreements and policies with publicly verifiable and authorized cryptographic proofs and Governance Automation, designed for DAOs. 
+Traditional methods of signing agreements, such as paper-based systems or legacy e-signatures, are prone to fraud and lack transparency, and offer no solution for verification of the signer's identity or signer’s authority to execute the agreement. Instead these systems rely on costly legal proceedings for enforce agreements. 
+
+DAOsign provides publicly verifiable cryptographic proofs; secure, transparent and automated processing of complex Agreements and Policies. 
+
+Furthermore, DAOsign provides robust Governance and Policy Builder platform to provide governance, workflow automation and controls for organization policies and procedures.
+### Project Details
+A project goal is to store Agreements Proofs on Polkadot chain using Ink! smart contracts. 
+
+
+Deliveries include DAOsign platform with an ability to:
+- create agreements including generating using ChatGPT
+- add and remove signers and observers of different levels of anonymity
+- allow signing the documents with further storing of signature proofs on Polkadot blockchain.
```

Comment:
This statement is a bit confusing because the relay chain itself doesn't run smart contracts. I'm assuming you mean that the proofs would be stored on whatever parachain you end up deploying to (testnet for now obviously).

Reply:
Thank you for pointing to it. Updated docs (we will deploy to Astar testnet). Is it ok?

Reply:
Yep, we don't really care about which one is used, I just want the language in the application to be correct. Thanks!